### ISSUE TITLE
Issues with Lists in Slate Transform

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/markdown-transform#109]
**Describe the bug**
The slate transform seems to swallow lists in some cases. Here is a sample test.

[list-slate.zip](https://github.com/accordproject/markdown-transform/files/3790312/list-slate.zip)

**update** Markdown to Slate seems to lose any structure except for the first paragraph:
```
bash-3.2$ cat list.md 
1. item one
2. item two
   - sublist
   - sublist

bash-3.2$ markus parse --sample list.md --slate --output data.json 
13:16:40 - info: Creating file: data.json
13:16:40 - info:
{
  "object": "value",
  "document": {
    "object": "document",
    "data": {
    },
    "nodes": [
      {
        "object": "block",
        "type": "ol_list",
        "data": {
          "tight": "true",
          "start": "1",
          "delimiter": "period"
        },
        "nodes": [
          {
            "object": "block",
            "type": "list_item",
            "data": {
            },
            "nodes": [
              {
                "object": "text",
                "text": "item one",
                "marks": []
              }
            ]
          }, 
          {
            "object": "block",
            "type": "list_item",
            "data": {
            },
            "nodes": [
              {
                "object": "text",
                "text": "item two",
                "marks": []
              }
            ]
          }
        ]
      }
    ]
  }
}
```

Which means normalization returns only the top level list:
```
bash-3.2$ markus draft --data data.json --slate 
13:18:14 - info: 
1. item one
2. item two
bash-3.2$ markus normalize --sample list.md --slate
13:18:31 - info: 
1. item one
2. item two
```

This is true of anything which isn't pure text from a single paragraph (breaks, nested lists, multiple chunks of text, etc.)

**Comment by jeromesimeon** [comment]
Proposed fix in #123 

Here is a test with nested lists:
```
bash-3.2$ cat test/data/nested-list.md 
Prolog
1. item one
2. item two
   - sublist
   - sublist

Epilog
bash-3.2$ ../markdown-cli/index.js normalize --sample test/data/nested-list.md --slate
16:45:40 - info: 
Prolog
1. item one
2. item two
   - sublist
   - sublist

Epilog
```

**Comment by jeromesimeon** [comment]
Fixed in #123
